src/components/PuzzleTitle/PuzzleTitle.js

```diff
@@ -0,0 +1,34 @@
+import { h } from 'preact';
+
+const textShadow = '0 1px 4px rgba(0, 0, 0, 0.15)';
+const margin = '0 0.3em 0 0;';
+
+const PuzzleTitle = ({ index, title }) => (
+    <p>
+        <span className="puzzle-index">#{index}</span>
+        <span className="puzzle-title">{ title }</span>
+
+        <style jsx>{`
+            p {
+                font-size: 1.5em;
```

Comment:
Maybe it's better to use `rem`s here so that `PuzzleTitle`'s font size doesn't depend on that of the containing component? Unless that's actually what we want.